diff --git a/runcoms/zpreztorc b/runcoms/zpreztorc
index 28b6005..a5b0234 100644
--- a/runcoms/zpreztorc
+++ b/runcoms/zpreztorc
@@ -35,7 +35,8 @@ zstyle ':prezto:load' pmodule \
   'spectrum' \
   'utility' \
   'completion' \
-  'prompt'
+  'prompt' \
+  'git' \
 
 #
 # Autosuggestions
diff --git a/runcoms/zshrc b/runcoms/zshrc
index 039b882..39468bc 100644
--- a/runcoms/zshrc
+++ b/runcoms/zshrc
@@ -11,3 +11,8 @@ if [[ -s "${ZDOTDIR:-$HOME}/.zprezto/init.zsh" ]]; then
 fi
 
 # Customize to your needs...
+export LSCOLORS=gxBxhxDxfxhxhxhxhxcxcx
+
+zstyle ':prezto:module:editor:info:keymap:primary' format ' %B%F{1}❯%F{8}❯%f%b'
+
+eval "$(rbenv init -)"
